README.md

```diff
@@ -36,12 +36,11 @@ or online at
 Documentation
 -------------
 
-If you downloaded an official release, the documentation can be found in $VIGRA_PATH/doc/vigra/, the start file 
-is $VIGRA_PATH/doc/vigra/index.html. Online documentation for the latest release is at 
-http://hci.iwr.uni-heidelberg.de/vigra/doc/vigra/index.html
+If you downloaded an official release, the documentation can be found in `$VIGRA_PATH/doc/vigra/`, the start file 
+is `$VIGRA_PATH/doc/vigra/index.html`.
 
 When you use the development version from github, you can generate documentation by `make doc`. Up-to-date 
-online documentation for the 'master' branch is at http://ukoethe.github.io/vigra/doc/vigra/index.html
+online documentation for the 'master' branch is regularly pushed to http://ukoethe.github.io/vigra/doc/vigra/
```

Comment:
If you place the encrypted password into the sources, anyone can read it, right? So it depends on what the password can do. If one creates a special account only for the docs, danger might be limited to the docs, but that would still mean that anyone could insert malicious code into the web pages. Does not sound like a good idea. How about letting any Linux machine run "git fetch && make doc && ..." in a cron job?